packages/fxa-auth-server/pm2.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   apps: [
     {
       name: 'auth',
-      script: 'ts-node bin/key_server.js',
+      script: 'node -r ts-node/register bin/key_server.js',
```

Comment:
What's the difference between `ts-node` and `node -r ts-node/register`?

Reply:
Ah [this](https://stackoverflow.com/a/53802296/717633) looks like a decent reason.

Reply:
it's a quirk of the shell env differences with yarn. With yarn at the time this line executes `ts-node` isn't in the `PATH`